DeviceTests/DeviceTests.Shared/HardwareSupport.cs

```diff
@@ -74,5 +74,16 @@ static class HardwareSupport
             // TODO: most UWP devices don't have a camera lamp (mobile devices do, we we don't care about those)
             false;
 #endif
+
+        public static bool HasBarometer =>
+#if __ANDROID__
+      true;
+#elif __IOS__
+            // iphone 6 and never have a barometer. looking in how to test this.
```

Comment:
// iHhone 6 and newer have a barometer. You can test the type of device too with the Model type technically.